src/Serilog.Sinks.Elasticsearch/Sinks/ElasticSearch/ElasticSearchSink.cs

```diff
@@ -242,7 +242,7 @@ private void HandleResponse(IEnumerable<LogEvent> events, DynamicResponse result
                             // Send to a failure callback
                             try
                             {
-                                _state.Options.FailureCallback(e);
+                                _state.Options.FailureCallback(e, null);
```

Comment:
We should find a way to raise a meaningful exception here as well. I did a test with 403 response from elastic and ended up here but the `null` doesn't tell anything to the higher-up code.
This gives me usable information to act upon (copied selflog from line 216):
```csharp
 _state.Options.FailureCallback(e, new Exception(string.Format("Failed to store event with template '{0}' into Elasticsearch. Elasticsearch reports for index {1} the following: {2}",
                                e.MessageTemplate,
                                action["_index"],
                                _state.Serialize(action["error"]))));
```